src/apiCalls.js

```diff
@@ -62,3 +62,21 @@ export const updateRatings = (movieId, rating, userId) => {
     return response.json();
   });
 };
+
+export const deleteRating = (ratingId, userId) => {
+  const options = {
+    method: 'DELETE',
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  };
+
+  return fetch(
+    `https://rancid-tomatillos.herokuapp.com/api/v1/users/${userId}/ratings/${ratingId}`,
+    options
+  ).then(response => {
+    if (!response.ok) {
+      throw Error('Problem with delete rating')
+    }
+  })
+}
```

Comment:
Hey all, I was running into errors with json(), so I cut out the complicating portions of the fetch request. You'll notice that the .then() statement here only includes logic for the case of a problematic response since the response body has no actual content to parse.